style/index.scss

```diff
@@ -7,6 +9,27 @@
 html {
   font-size: 10px;
 }
+body {
+  &::-webkit-scrollbar {
+    width: 6px;
+  }
+
+  /* Track */
+  &::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px $gray;
+  }
+
+  /* Handle */
+  &::-webkit-scrollbar-thumb {
+    background: #9c3;
```

Comment:
Make a variable for this color.... 
note:- We don't need to create a new branch for this new layout change just do this in this same brnach.

Reply:
But, You said we need to create a new branch for every new things